index.js

```diff
@@ -1,9 +1,19 @@
 function calculate(numberOne, operation, numberTwo) {
+  if (typeof numberOne === 'undefined') {
+    numberOne = 0
+  }
+  if (typeof numberTwo === 'undefined') {
+    numberTwo = 0
+  }
   switch (operation) {
     case '+':
-      return numberOne - numberTwo
+      return parseInt(numberOne) + parseInt(numberTwo)
```

Comment:
Nice job on the parseInt to avoid the concatenation and ensure numbers are used.